website/pages/docs/extending/plugins.mdx

```diff
@@ -34,18 +34,18 @@ of Packer starts and communicates with.
 
 These plugin applications aren't meant to be run manually. Instead, Packer core
 executes them as a sub-process, run as a sub-command (`packer plugin`) and
-communicates with them. For example, the VMware builder is actually run as
-`packer plugin packer-builder-vmware`. The next time you run a Packer build,
+communicates with them. For example, the Shell provisioner is actually run as
+`packer plugin packer-provisioner-shell`. The next time you run a Packer build,
 look at your process list and you should see a handful of `packer-` prefixed
 applications running.
 
 ## Installing Plugins
 
 The easiest way to install a plugin is to name it correctly, then place it in
 the proper directory. To name a plugin correctly, make sure the binary is named
-`packer-TYPE-NAME`. For example, `packer-builder-amazon-ebs` for a "builder"
-type plugin named "amazon-ebs". Valid types for plugins are down this page
-more.
+`packer-TYPE-NAME`. For example, `packer-plugin-amazon` for a "multi-plugin"
```

Comment:
Since the future state will be `packer-plugin-*` I think `packer-TYPE-NAME` should be updated. 
```suggestion
`packer-plugin-NAME`. For example, `packer-plugin-amazon` for a "plugin"
```

Reply:
Yup I hesitated about doing that and didn't. But I think you're right !